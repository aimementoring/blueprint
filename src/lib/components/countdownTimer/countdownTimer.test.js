import React from 'react';
import renderer from 'react-test-renderer';
import CountdownTimer from './countdownTimer.js';

describe('CountdownTimer', () => {
  it('renders properly with no date', () => {
    const tree = renderer.create(<CountdownTimer />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders properly with custom date', () => {
    const tree = renderer.create(<CountdownTimer date="2019-10-08T09:00:00" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
