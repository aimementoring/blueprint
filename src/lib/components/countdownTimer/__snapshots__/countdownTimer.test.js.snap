// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CountdownTimer renders properly with custom date 1`] = `
<div
  className="theme-base"
>
  <div
    className="countDownContainerFeatureBoard"
  >
    <div
      className="countdown"
    >
      <span
        className="countdownCol"
      >
        <span
          className="countdownColElement"
        >
          <span
            className="countdownNumber"
          >
            00
          </span>
          <span
            className="countdownLabel"
          >
            Years
          </span>
        </span>
      </span>
      <span
        className="countdownCol"
      >
        <span
          className="countdownColElement"
        >
          <span
            className="countdownNumber"
          >
            00
          </span>
          <span
            className="countdownLabel"
          >
            Days
          </span>
        </span>
      </span>
      <span
        className="countdownCol"
      >
        <span
          className="countdownColElement"
        >
          <span
            className="countdownNumber"
          >
            00
          </span>
          <span
            className="countdownLabel"
          >
            Hours
          </span>
        </span>
      </span>
      <span
        className="countdownCol"
      >
        <span
          className="countdownColElement"
        >
          <span
            className="countdownNumber"
          >
            00
          </span>
          <span
            className="countdownLabel"
          >
            Mins
          </span>
        </span>
      </span>
      <span
        className="countdownCol"
      >
        <span
          className="countdownColElement"
        >
          <span
            className="countdownNumber"
          >
            00
          </span>
          <span
            className="countdownLabel"
          >
            Secs
          </span>
        </span>
      </span>
    </div>
  </div>
</div>
`;

exports[`CountdownTimer renders properly with no date 1`] = `
<div
  className="theme-base"
>
  <div
    className="countDownContainerFeatureBoard"
  >
    <div
      className="countdown"
    >
      <span
        className="countdownCol"
      >
        <span
          className="countdownColElement"
        >
          <span
            className="countdownNumber"
          >
            00
          </span>
          <span
            className="countdownLabel"
          >
            Years
          </span>
        </span>
      </span>
      <span
        className="countdownCol"
      >
        <span
          className="countdownColElement"
        >
          <span
            className="countdownNumber"
          >
            00
          </span>
          <span
            className="countdownLabel"
          >
            Days
          </span>
        </span>
      </span>
      <span
        className="countdownCol"
      >
        <span
          className="countdownColElement"
        >
          <span
            className="countdownNumber"
          >
            00
          </span>
          <span
            className="countdownLabel"
          >
            Hours
          </span>
        </span>
      </span>
      <span
        className="countdownCol"
      >
        <span
          className="countdownColElement"
        >
          <span
            className="countdownNumber"
          >
            00
          </span>
          <span
            className="countdownLabel"
          >
            Mins
          </span>
        </span>
      </span>
      <span
        className="countdownCol"
      >
        <span
          className="countdownColElement"
        >
          <span
            className="countdownNumber"
          >
            00
          </span>
          <span
            className="countdownLabel"
          >
            Secs
          </span>
        </span>
      </span>
    </div>
  </div>
</div>
`;
